core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -455,7 +455,7 @@ public function showAll()
 <div class="tl_formbody_edit">
 <input type="hidden" name="FORM_SUBMIT" value="tl_select">
 <input type="hidden" name="REQUEST_TOKEN" value="'.REQUEST_TOKEN.'">' : '').($blnClipboard ? '
-<div id="paste_hint">
+<div id="paste_hint" data-additional-scroll-offset-relevant>
```

Comment:
Do you think we can short this a little bit? 🙈

Reply:
I'm not a fan of nonsense abbreviations like `asor`. Nobody will know what it does. Proposals?

Reply:
Maybe `data-add-to-scroll-offset`? That would still be clear and 13 characters shorter.

Reply:
Done.